### PR TITLE
group add tailwind/已解衝突

### PR DESCRIPTION
**Pull request by KathyLEEKUANYI**

### CODE REVIEW

File: app/views/groups/content.html.erb [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
-      <div>
-        描述<%= @group.description %>
-        <%= @group.id %>
+<section  class="w-full h-full fixed left-0 top-0 mx-auto flex justify-center items-center bg-black bg-opacity-30">
+  <div class="w-1/3 h-3/5 p-2.5 bg-white rounded-3xl">
+    <div class="align-center justify-between flex">
+      <span class="my-6 text-2xl text-gray-900 font-semibold ml-6"><%= @group.title %></span>
+      <span class="my-6 text-gray-400 text-2xl mr-6 bg-white hover:bg-gray-100 py-1 px-2 rounded-lg "><%= link_to "X", @group %></span>
+    </div>
+      <div class="my-2 text-2xl ml-6">
```
Comment (cacachang) [inline app/views/groups/content.html.erb:7]: 縮排沒改

File: app/views/groups/edit.html.erb [hunk 2]
```diff
@@ -1,13 +1,18 @@
...
-
-<%= form_for @group do |form| %>
-  <div>
-    名稱：<%= form.text_field :title %>
+<section  class="w-full h-full fixed left-0 top-0 mx-auto flex justify-center items-center bg-black bg-opacity-30">
+  <div class="w-1/3 h-3/5 p-2.5 bg-white rounded-3xl bg-opacity-80">
+    <h3 class="my-6 text-2xl text-gray-900 font-semibold">編輯說明</h3>
+    <%= form_for @group do |form| %>
+     <div class="text-gray-700 font-bold py-2">名稱:</div>
+      <div>
```
Comment (cacachang) [inline app/views/groups/edit.html.erb:6]: 這裡不用縮吧？

File: app/views/groups/new.html.erb [hunk 3]
```diff
@@ -1,19 +1,23 @@
...
-
-    <p>频道是你的團隊進行通信的地方。圍繞主题進行組織時其效果最佳，例如#市場營銷#。</p>
-
+<section class="w-full h-full fixed left-0 top-0 mx-auto flex justify-center items-center bg-black bg-opacity-30">
+  <div class="w-1/3 h-3/5 p-2.5 bg-white rounded-3xl bg-white">
+    <div class="align-center justify-between flex">
+      <span class="my-6 text-2xl text-gray-900 font-semibold">創建頻道</span>
+      <span class="my-6 text-gray-400 text-2xl mr-6 bg-white hover:bg-gray-100 py-1 px-2 rounded-lg "><%= link_to "X", groups_path %></span>
+    </div>
+    <p class="my-6 text-gray-500">频道是你的團隊進行通信的地方。圍繞主题進行組織時其效果最佳，例如#市場營銷#。</p>
```
Comment (cacachang) [inline app/views/groups/new.html.erb:7]: 是說簡字何時要改掉～

File: app/views/groups/show.html.erb [hunk 4]
```diff
@@ -1,16 +1,89 @@
...
-    這裡是群組聊天對話的位置
-  <% else %>
-
-    <%= link_to "詳情" , content_group_path(@group.id),method:"post" %>
-    <%= link_to "加入頻道" , join_group_path(@group),method: "post" %>
-    <%= link_to "返回所有頻道" , groups_path %>
+    <div class="my-6 text-2xl text-blue-400 font-semibold ml-10 mt-8"><%= @group.description %><%= link_to "編輯說明" , edit_group_path(@group) %></div>
 
+    <div class="flex h-screen antialiased text-gray-800">
+    <div class="flex flex-row h-full w-full overflow-x-hidden">
```
Comment (cacachang) [inline app/views/groups/show.html.erb:11]: 縮排

**Review by cacachang — CHANGES_REQUESTED** [review]

File: app/views/groups/content.html.erb (hunk 1 above)
Comment (KathyLEEKUANYI) [reply app/views/groups/content.html.erb:7]: <div class="align-center justify-between flex">
      <span class="my-6 text-2xl text-gray-900 font-semibold ml-6"><%= @group.title %></span>
      <span class="my-6 text-gray-400 text-2xl mr-6 bg-white hover:bg-gray-100 py-1 px-2 rounded-lg "><%= link_to "X", @group %></span>
    </div>

這樣是一組

      <div class="my-2 text-2xl ml-6">
        <div class="text-gray-700 font-bold py-2">描述:</div>
        <div class="p-1 mt-5"><%= @group.description %></div>
      </div>

這是另一組

File: app/views/groups/edit.html.erb (hunk 2 above)
Comment (KathyLEEKUANYI) [reply app/views/groups/edit.html.erb:6]: <div class="text-gray-700 font-bold py-2">名稱:</div>

---------------------------
      <div>
        <%= form.text_field :title ,:class=>"text-gray-700 shadow border rounded border-gray-300 focus:outline-none focus:shadow-outline py-1 px-3 mb-3", placeholder:"名稱"  %>
      </div>

File: app/views/groups/new.html.erb (hunk 3 above)
Comment (KathyLEEKUANYI) [reply app/views/groups/new.html.erb:7]: 哪個是簡字？看起來沒有？）

**Review by AilaLu — APPROVED** [review]

**Pull request closed without merging**